### PR TITLE
[ENHANCEMENT] Added section 2 with container installation

### PR DESCRIPTION
**Pull request by eschabell**
# Description

Added section 2 and container installation with placeholder for source code install.

# Screenshots

None.

# Checklist

- [ ] Pull request has a descriptive title and context useful to a reviewer.
- [ ] Pull request title follows the `[<catalog_entry>] <commit message>` naming convention using one of the following `catalog_entry` values: `FEATURE`, `ENHANCEMENT`, `BUGFIX`, `BREAKINGCHANGE`, `IGNORE`.
- [ ] All commits have [DCO signoffs](https://github.com/probot/dco#how-it-works).

## UI Changes

- [ ] Changes that impact the UI include screenshots and/or screencasts of the relevant changes.
- [ ] Code follows the [UI guidelines](https://github.com/perses/perses/blob/main/ui/ui-guidelines.md).
- [ ] Visual tests are stable and unlikely to be flaky. See [Storybook](https://github.com/perses/perses/tree/main/ui/storybook#visual-tests) and [e2e](https://github.com/perses/perses/tree/main/ui/e2e#visual-tests) docs for more details. Common issues include:
  - Is the data inconsistent? You need to mock API requests.
  - Does the time change? You need to use consistent time values or mock time utilities.
  - Does it have loading states? You need to wait for loading to complete.

### CODE REVIEW

**Review by karsov — COMMENTED** [review]
I have a few suggestions/topics for discussion related to this PR:

1. What do you think about adding a markdown linter? I like the quite popular [markdownlint](https://github.com/DavidAnson/markdownlint). To me it makes the raw files more readable and the output more user-friendly.
2. Docker is quite more popular than podman. What do you think about making the instructions more general so they cover both tools (the commands are basically the same)?
3. To me it seems that "Perses Easy install" is outside the scope of the "Installation" instructions. If we want to reference it for further reading we can put a link. Copying the instructions from another repo for using another repo's tools makes things more prone to drift and inconsistency (and errors in general) with time. With time the Perses repo should be the source of truth for documentation and not an external repo.

P.S. I can work on 1 & 2 in a different PR if there is a general agreement.

File: docs/user/section-2-podman.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+$ podman run --name perses -d -p 127.0.0.1:8080:8080 persesdev/perses:v{VERSION-NUMBER}
+```
+## Connect a browser (default)
+
+Open the Perses console at http://localhost:8080.
+
+You are presented with an empty home dashboard, in light mode. For fun you can choose optionally to flip the switch in
+the top right corner to enable dark mode.
+
+## But this version is empty?
```
Comment (Nexucis) [inline docs/user/section-2-podman.md:55]: I agree with @karsov, these parts shouldn't be in a user documentation.

On the contrary starting from scratch will give the possibility to introduce various notion, such as datasource / global variable, global secrets and of course dashboard.

That's a user documentation, so as a user I'm expecting to find information about how to install it of course, but after how I can use the UI, the CLI, the data provisioning potentially etc. 

Basically we should find information about any features Perses is providing.

And then if there is a third party that proposes additional content around Perses then yeah we can reference it if it makes sense.

Comment (eschabell) [reply docs/user/section-2-podman.md:55]: This is all coming in the following sections, 3 and 4 specifically, after installation. This is just to get something up easily and ready for user to explore. After showing the API, CLI, UI, etc, then user designs a simple dashboard, then adds to that, and finally ends up with a complex dashboard.

File: docs/user/section-2-podman.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+
+```shell
+$ podman machine start
+```
+
+## Running container image
+It's pretty straight forward to running Perses in a container, just start the Perses container image as follows:
+
+```shell
+$ podman run --name perses -d -p 127.0.0.1:8080:8080 persesdev/perses:v{VERSION-NUMBER}
```
[the remark this answers is not in the document]
Comment (eschabell) [reply docs/user/section-2-podman.md:27]: The problem I've found with latest is it's hard to keep up with... so by using a placeholder, it does not require as much document maintenance to keep relevant for teaching the new user. Make sense?

**Comment by Gladorme** [comment]
IMO, this Perses Easy Install has nothing to do here: it's coming from a third-party repo. The installation doc with Docker/Podman and from source should be enough. 
If we want to provide initialized data we could include a script to populate easily, but not something that is encapsulating Perses binaries on a third-party repo 🤔

File: docs/user/section-2-podman.md [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+```
+
+The details in this command are that we give the container a referencable name (--name perses), detach the container 
+from the command line (-d), map the local machine port 8080 to the container port 8080 (-p 127.0.0.1:8080:8080), and use 
+the image version supported in these instructions (persesdev/perses:v{VERSION-NUMBER}). Note: you can use any local port you have 
+available, but you need to map to the container port 8080.
+
+### Note: What do do with failures?
```
[the remark this answers is not in the document]
Comment (eschabell) [reply docs/user/section-2-podman.md]: Fixed #3 by removing the external project from docs.

File: docs/user/section-2-podman.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (eschabell) [reply docs/user/section-2-podman.md:27]: Fixed by using latest.

**Comment by eschabell** [comment]
Removed third party repo project.

File: docs/user/section-2-podman.md (hunk 1 above)
Comment (eschabell) [reply docs/user/section-2-podman.md]: Removed usage of install project.

File: docs/user/section-2-podman.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
+# Section 2 - Installing Perses in a Container
+
+This sections continues with installing using a container image with Podman open source container tooling, configure 
+examples, and run the container on your local machine.
+
+## Installing Podman tooling
+
+Installing Perses using a container image is going to be demonstrated here using the 
+[open source project Podman](https://podman.io/). It's assumed you have already installed the Podman commandline tooling 
+previously. For those using Docker, you'll find examples for running this tooling too.
```
[the remark this answers is not in the document]
Comment (eschabell) [reply docs/user/section-2-podman.md]: Added this as a fix, did it from the forked branch to ensure sign-off on the commit (accepted other commits here and they broke the build pipeline without the sign-off.)

File: docs/user/section-2-podman.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
+# Section 2 - Installing Perses in a Container
```
[the remark this answers is not in the document]
Comment (eschabell) [reply docs/user/section-2-podman.md:1]: Done.

**Review by Nexucis — APPROVED** [review]
nice. Thank you @eschabell for continuing this work!
And thank you @karsov for all your comments and suggestions !!
